6.2.строковый_тип_данных/6.2.5.py

```diff
@@ -0,0 +1 @@
+print('''"Python is a great language!", said Fred. "I don't ever remember having this much fun before."''')
```

Comment:
Конечно можно так, но я сделал вот так:
```suggestion
print("\"Python is a great language!\", said Fred. \"I don't ever remember having this much fun before.\"")
```

Reply:
Ну тут и там свои плюс и минусы для читаемости